<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_42</class>
 <widget class="QDialog" name="window_42">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="SelectText_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>곧 도라온</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
    </font>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>330</y>
     <width>421</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>               시 까지 잠시 자리를 비웁니다.</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentTime_2">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>330</y>
     <width>80</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="CurrentID">
      <property name="font">
       <font>
        <family>HY견고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>HY견고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>님은</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="CurrentSeat">
      <property name="font">
       <font>
        <family>HY견고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>HY견고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>번 좌석이</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="CurrentTime">
      <property name="font">
       <font>
        <family>HY견고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>HY견고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>시까지 예약되어있습니다.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
